DataFormats/TotemDigi/src/classes_def.xml

```diff
@@ -15,7 +15,7 @@
   <class name="edm::Wrapper<TotemTriggerCounters>"/>
 
   <class name="TotemVFATStatus" ClassVersion="2">
-    <version ClassVersion="2" checksum="3003040825"/>
+    <version ClassVersion="2" checksum="1439138260"/>
```

Comment:
we didn't make anything yet with version=2. So, this one is OK.
For the future, when the code is running, the version number should be increased